source/quickstart.txt

```diff
@@ -307,35 +307,3 @@ Synchronization Notes
 
 - To estimate the size of ``oplog`` needed for initial synchronization,
   see :ref:`c2c-oplog-sizing`.
-
-.. _c2c-quickstart-many-with-one:
```

Comment:
Deleted this from the quickstart because C2C engineering wanted us to keep beta features only within the beta features TOC drawer and not in the GA docs.